packages/components/src/presentation-components/instance-filter-builder/Utils.ts

```diff
@@ -202,24 +202,49 @@ function quantityPropertyValidator({ property, value }: ValidatorContext) {
   return undefined;
 }
 
-function numericPropertyValidator({ property, value }: ValidatorContext) {
+function numericPropertyValidator({ property, value, operator }: ValidatorContext) {
   if (!isPropertyNumeric(property.typename) || value === undefined) {
     return undefined;
   }
 
-  if (isInvalidNumericValue(value)) {
+  if (isInvalidNumericValue(value) && (!isOperatorValidForSerialization(operator) || isInvalidSerializedNumericValue(value))) {
```

Comment:
Looking at this condition I think `numericPropertyValidator` should skip `Equal` and `NotEqual` operators. For those operators we show unique values selector. I don't think it makes sense to show not a number error message for unique values selector because user cannot edit values shown and if some of those values are not valid numbers than there is some problems in our code that makes us load invalid values for numeric properties.

Reply:
That makes a lot of sense, will adjust